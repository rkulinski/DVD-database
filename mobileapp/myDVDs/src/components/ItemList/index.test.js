import React from 'react';
import { shallow } from 'enzyme';
import { ItemList } from './';


describe('Testing Header component', () => {
  it('renders as expected', () => {
    const wrapper = shallow(
      <ItemList/>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
