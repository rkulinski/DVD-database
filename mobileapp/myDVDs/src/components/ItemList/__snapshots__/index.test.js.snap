// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Header component renders as expected 1`] = `
<TouchableOpacity
  activeOpacity={0.2}
>
  <Component
    style={
      Array [
        Object {
          "alignItems": "center",
          "flex": 1,
          "flexDirection": "row",
          "paddingBottom": 15,
          "paddingLeft": 10,
          "paddingRight": 10,
          "paddingTop": 15,
        },
      ]
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "fontSize": 37.5,
          },
        ]
      }
    />
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "marginLeft": "auto",
            "paddingRight": 10,
          },
        ]
      }
    >
      ‚ùØ
    </Text>
  </Component>
</TouchableOpacity>
`;
